pep-9999.rst

```diff
@@ -47,13 +47,47 @@ required to interpret all standard constructs, and additionally a type checker
 can support extra constructs not described in this PEP. All deviations from
 this PEP should be documented.
 
+Syntax
+======
+
+Type stubs are syntactically valid Python files with a ``.pyi`` suffix.
+They should be valid according to the latest Python version six months
+after release at the latest, but can start to use features from that
+release from the release date on.
+Type stubs authors are encouraged to use the latest available syntax
+features in stubs, even if the type information in the stub is targeted
+to an older Python version.
+
+Type checkers are expected to support the syntax of the latest released Python
+version.
+
+For example, Python 3.7, which enabled the new keywords `async`
+and `await` (see [#pep492]_), was released on June 27nd, 2018. Type
+checkers should support them by that date and type stubs could start
+using `async` to mark function definitions. On the other hand, type stubs
+should stop using the new keyword as an attribute or function name before
+December 27nd, 2018.
```

Comment:
This is actually a bit of a white lie. I think `async` could be used before that date (since it was introduced in Python 3.5), but I think as an example that is fine.

Reply:
They're good examples because `async` and `await` became reserved keywords in 3.7, so using them as anything but reserved keywords in a stub file could be an error once the 6 month grace period ends.

I would change "On the other hand" to "Additionally" though.